VirtoCommerce.CatalogModule.Data/Services/CatalogSearchServiceDecorator.cs

```diff
@@ -54,7 +54,7 @@ public virtual SearchResult Search(SearchCriteria criteria)
                 serviceCriteria.ObjectType = KnownDocumentTypes.Product;
                 serviceCriteria.SearchPhrase = criteria.Keyword;
                 serviceCriteria.CatalogId = criteria.CatalogId;
-                serviceCriteria.Outline = criteria.CategoryId;
+                serviceCriteria.Outline = criteria.Outline;
```

Comment:
Need to add backward compatibility with existing clients that still use CategoryId:
`serviceCriteria.Outline = criteria.Outline ?? criteria.CategoryId`